app/models/game.rb

```diff
@@ -23,7 +23,6 @@ def reset_piece_players
     pieces.where(is_black: false).update_all(user_id: user_id_white)
   end
 
-  # rubocop:disable Metrics/AbcSize
```

Comment:
Although actually the error isn't for move! any more, it's for:
app/models/game.rb:27:3: C: Assignment Branch Condition size for initialize_board is too high. [56/15]
  def initialize_board